ts/features/bonus/bpd/screens/onboarding/declaration/DeclarationEntry.tsx

```diff
@@ -13,6 +13,7 @@ const styles = StyleSheet.create({
 type Props = {
   // in order to accepts composite text with bold
   text: XOR<string, React.ReactNode>;
+  onValueChange?: (value: boolean) => void;
 };
 
 type OwnProps = Props & React.ComponentProps<typeof CheckBox>;
```

Comment:
If we add `  onValueChange?: (value: boolean) => void;` as props, we don't need anymore  React.ComponentProps<typeof CheckBox> (or viceversa)
```suggestion
type OwnProps = Props;
```